embedded-graphics/src/fonts/text.rs

```diff
@@ -115,7 +116,11 @@ where
     /// [`Size::zero()`]: ../geometry/struct.Size.html#method.zero
     fn size(&self) -> Size {
         let width = if !self.primitive.text.is_empty() {
-            (F::CHARACTER_SIZE.width + F::CHARACTER_SPACING) * self.primitive.text.len() as u32
+            self.primitive
+                .text
+                .chars()
+                .map(|c| F::char_width(c) + F::CHARACTER_SPACING)
+                .sum::<u32>()
                 - F::CHARACTER_SPACING
```

Comment:
Thanks for finding this off-by-one :grin:

Reply:
:smile: